lib/membrane_h264_ffmpeg/decoder.ex

```diff
@@ -10,7 +10,15 @@ defmodule Membrane.H264.FFmpeg.Decoder do
   alias __MODULE__.Native
   alias Membrane.Buffer
   alias Membrane.Caps.Video.{H264, Raw}
-  use Bunch
+  alias Membrane.H264.FFmpeg.Common
+
+  def_options add_pts?: [
+                spec: boolean(),
+                default: false,
+                description: """
+                Setting this flag to `true` causes decoder to add presentation timestamp (pts) taken from buffer timestamp into the AVPacket and in consequence to the produced frame.
+                """
+              ]
```

Comment:
I've missed that before, but why is this configurable? I don't really see the point of not passing PTS, they can be ignored after all